wear/src/main/java/com/eveningoutpost/dexdrip/Models/UserNotification.java

```diff
@@ -179,27 +179,36 @@ public static UserNotification create(String message, String type, long timestam
         UserNotification userNotification = new UserNotification();
         userNotification.timestamp = timestamp;
         userNotification.message = message;
-        if (type == "bg_alert") {
-            userNotification.bg_alert = true;
-        } else if (type == "calibration_alert") {
-            userNotification.calibration_alert = true;
-        } else if (type == "double_calibration_alert") {
-            userNotification.double_calibration_alert = true;
-        } else if (type == "extra_calibration_alert") {
-            userNotification.extra_calibration_alert = true;
-        } else if (type == "bg_unclear_readings_alert") {
-            userNotification.bg_unclear_readings_alert = true;
-        } else if (type == "bg_missed_alerts") {
-            userNotification.bg_missed_alerts = true;
-        } else if (type == "bg_rise_alert") {
-            userNotification.bg_rise_alert = true;
-        } else if (type == "bg_fall_alert") {
-            userNotification.bg_fall_alert = true;
-        } else {
-            Log.d(TAG,"Saving workaround for: "+type+" "+message);
-            Pref.setString("UserNotification:timestamp:" + type, JoH.qs(timestamp));
-            Pref.setString("UserNotification:message:" + type, message);
-           return null;
+        switch (type) {
```

Comment:
Are you certain all of these have been transcribed correctly? Was this done with an automated tool?

Reply:
This was done automatically by AndroidStudio.

Reply:
@jamorham Added test for this class in pull request 376